### PR TITLE
Backbone 1.1.0 compatibility fixes

### PR DESCRIPTION
**Pull request by hijonathan**
Backbone 1.1.0 no longer assigns the view arguments to `this.options`. Instead, it enforces `pick` to filter out non-whitelisted keyword arguments, but it still passes them to the view's `initialize` method.

Not sure if this gets all cases. Tests (ahem) would help.

### CODE REVIEW

**Comment by hijonathan** [comment]
As a side note, I version locked stuff because rivets `0.6.x` seemed to break this too.

**Comment by ajb** [comment]
Let's merge this in when we have tests.

File: coffee/main.coffee [hunk 1]
```diff
@@ -387,9 +391,10 @@ class Formbuilder
 
             @updatingBatch = undefined
 
-  constructor: (selector, opts = {}) ->
+  constructor: (opts={}) ->
     _.extend @, Backbone.Events
-    @mainView = new Formbuilder.views.main _.extend({ selector: selector }, opts, { formBuilder: @ })
+    args = _.extend({ selector: opts.selector }, opts, { formBuilder: @ })
```
Comment (hijonathan) [inline coffee/main.coffee:396]: In hindsight, this is actually redundant. Should be simplified to:

```
opts.formBuilder = @
```

Unless it's actually doing a deep extend, in which case:

```
_.extend opts, {formBuilder: @}
```

**Comment by hijonathan** [comment]
Shutting down my fork and dealing with this in the [bb-1.1.0](https://github.com/dobtco/formbuilder/tree/bb-1.1.0) branch. Will open a new PR when ready.

**Pull request closed without merging**